SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorModule.cpp

```diff
@@ -75,6 +79,14 @@ void FSpatialGDKEditorModule::ShutdownModule()
 	}
 }
 
+void FSpatialGDKEditorModule::TakeSnapshot(UWorld* World, FSpatialSnapshotTakenFunc OnSnapshotTaken)
+{
+	bool bUseStandardRuntime =
```

Comment:
I guess you could change it mid-run and this would fail, but that is a rather rare use-case.

Reply:
This would be very non-kosher imo. Also given that Kraken is going away, this should all be temporary code.